sdk/cdsclient/http.go

```diff
@@ -186,6 +187,8 @@ func (c *client) Stream(method string, path string, body io.Reader, noTimeout bo
 			continue
 		}
 
+		req = req.WithContext(ctx)
```

Comment:
it could be nice to propagate tracing context in the request:
https://github.com/ovh/cds/blob/1365b53abab9328fd0742d03e34f4f2598f97af5/engine/api/services/http.go#L166

Reply:
done